src/Application/Computation/Model.cs

```diff
@@ -6,26 +6,27 @@
 
 namespace Symbolica.Computation;
 
-internal sealed class Model : IModel
+internal sealed class Model<TContext> : IModel
```

Comment:
What's the generic needed for?

Reply:
Oops. I moved it now. I think the rest are where they need to be.